squad/settings.py

```diff
@@ -170,7 +172,12 @@
 }
 database_config = os.getenv('DATABASE')
 if database_config:
-    db_from_env = dict(x.split('=') for x in database_config.split(':'))
+    db_from_env = dict()
+    for item in database_config.split(';'):
```

Comment:
could you make this split by `:` instead of `;`?
This way current setups wouldn't need to change their environment variables if they didn't need to pass extra settings